src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    private boolean isActive = false;
```

Comment:
by default boolean variable has `false` value, so we can remove initialization.

Reply:
It was removed.